embassy-stm32/src/usb/mod.rs

```diff
@@ -26,6 +27,7 @@ fn common_init<T: Instance>() {
     // Clock might not be exact 48Mhz due to rounding errors in PLL calculation, or if the user
     // has tight clock restrictions due to something else (like audio).
     #[cfg(not(stm32h7rs))]
+    #[cfg(not(all(stm32u5, peri_usb_otg_hs)))]
```

Comment:
could you change this to `#[cfg(not(any(....)))]` ? for consistency with the other `cfg` above, and also because it's easier to read. having two`cfg`s can get confusing.

Reply:
Ack